src/DependencyInjection/SymfonyOpenTracingExtension.php

```diff
@@ -19,8 +19,23 @@ protected function loadInternal(array $mergedConfig, ContainerBuilder $container
         $loader = new YamlFileLoader($container, new FileLocator(__DIR__.'/../Resource/config'));
         $loader->load('services.yml');
 
-        $container->setParameter('symfony_open_tracing.service_name', $mergedConfig['service_name']);
-        $container->setParameter('symfony_open_tracing.enabled', $mergedConfig['enabled']);
-        $container->setParameter('symfony_open_tracing.tracer_config', $mergedConfig['tracer_config']);
+        if ($mergedConfig['enabled']) {
+            $this->configureJaegerTracer($container, $mergedConfig);
+        }
+
+        $container
+            ->getDefinition('open_tracing.http_listener')
+            ->setArgument('$skippedRoutes', $mergedConfig['http_listener_skipped_routes'] ?? []);
```

Comment:
`$mergedConfig['http_listener_skipped_routes']` will be `[]` (at least after suggestion above, that this fallback is not necassary)

Reply:
right, fixed!